src/ploomber/telemetry/telemetry.py

```diff
@@ -151,15 +248,17 @@ def check_uid():
             with uid_path.open("w") as file:
                 yaml.dump({"uid": uid}, file)
             return uid
-        except FileNotFoundError as e:
-            return f"ERROR: Can't read UID file: {e}"
+        except Exception as e:
+            print(f"ERROR: Can't write UID file: {e}")
+            return f"NO_UID {e}"
     else:  # read and return uid
         try:
             with uid_path.open("r") as file:
                 uid_dict = yaml.safe_load(file)
             return uid_dict['uid']
-        except FileNotFoundError as e:
-            return f"Error: Can't read UID file: {e}"
+        except Exception as e:
+            print(f"Error: Can't read UID file: {e}")
+            return f"NO_UID {e}"
```

Comment:
what is this value used for? may be better to return `None`

Reply:
Added a flag in metadata for missing uids